### PR TITLE
Integrate contract and validations in buy modal

### PR DESCRIPTION
**Pull request by medardm**

### CODE REVIEW

**Review by mativanarquero — APPROVED** [review]

**Review by Pseudoman21 — APPROVED** [review]
Integrated!

File: src/config/index.ts [hunk 1]
```diff
@@ -17,12 +17,12 @@ export const USDT = new Token(ChainId.MAINNET, '0x55d398326f99059ff7754852469990
 export const EOS = new Token(ChainId.MAINNET, '0x56b6fb708fc5732dec1afc8d8556423a2edccbd6', 18, 'EOS', 'EOS Token')
 export const DOT = new Token(ChainId.MAINNET, '0x7083609fce4d1d8dc0c979aab8c869ea2c873402', 18, 'DOT', 'Polkadot Token')
 export const ETH = new Token(ChainId.MAINNET, '0x2170ed0880ac9a755fd29b2688956bd959f933f8', 18, 'ETH', 'Ethereum Token')
+export const OWN = new Token(ChainId.MAINNET, '0x2170ed0880ac9a755fd29b2688956bd959f933f8', 18, 'OWN', 'Ownly Token') // to update
```
Comment (harveyjavier) [inline src/config/index.ts:20]: Test address man lng ini?

Comment (harveyjavier) [reply src/config/index.ts:20]: Binance-Peg Ethereum Token (ETH) palan yadi haha

**Review by kojiadrianojr — APPROVED** [review]
Integratified!

**Review by swengr-janan — APPROVED** [review]
AGITH!

**Review by harveyjavier — APPROVED** [review]
Amalgamated!

**Pull request merged**